inputs/manual_input.csv

```diff
@@ -256,3 +256,21 @@ Haber-Bosch,hydrogen-input,0,1.1484,MWh_H2/MWh_NH3,-,"DECHEMA 2017: DECHEMA: Low
+direct iron reduction furnace,hydrogen-input,2020,2.1,MWh_H2/t_hbi,2020,"Mission Possible Partnership (2022): Steel Model Documentation (https://mpp.gitbook.io/mpp-steel-model/model-overview/model-components/technologies, accessed: 2022-12-05). ","63 kg H2/t steel for process ‘DRI-EAF_100% green H2’ according to documentation (raw input files for MPP model list 73 kg H2 / t steel, which seems to high and is probably incorrect)."
+direct iron reduction furnace,ore-input,2020,1.59,t_ore/t_hbi,2020,"Mission Possible Partnership (2022): Steel Model (https://github.com/missionpossiblepartnership/mpp-steel-model/blob/9eca52db92bd2d9715f30e98ccaaf36677fdb516/mppsteel/data/import_data/Technology%20Business%20Cases.csv, accessed: 2022-12-03). ",Based on process ‘DRI-EAF_100% green H2’.
+electric arc furnace,investment,2020,1839600,EUR/t_steel/h,2020,"Model assumptions from MPP Steel Transition Tool: https://github.com/missionpossiblepartnership/mpp-steel-model/blob/9eca52db92bd2d9715f30e98ccaaf36677fdb516/mppsteel/data/import_data/CAPEX%20OPEX%20Per%20Technology.xlsx, accessed: 2022-12-05.",210 EUR/t_steel output/a. MPP steel tool uses CAPEX/OPEX for technology ‘EAF’.
+electric arc furnace,FOM,2020,30,%/year,2020,"Model assumptions from MPP Steel Transition Tool: https://github.com/missionpossiblepartnership/mpp-steel-model/blob/9eca52db92bd2d9715f30e98ccaaf36677fdb516/mppsteel/data/import_data/CAPEX%20OPEX%20Per%20Technology.xlsx, accessed: 2022-12-05.","EAF has high OPEX of 62.99 EUR/year/t_steel, presumably because of electrode corrosion."
+electric arc furnace,lifetime,2020,40,years,2020,"Model assumptions from MPP Steel Transition Tool: https://github.com/missionpossiblepartnership/mpp-steel-model/blob/9eca52db92bd2d9715f30e98ccaaf36677fdb516/mppsteel/data/import_data/CAPEX%20OPEX%20Per%20Technology.xlsx, accessed: 2022-12-05.",MPP steel model distinguishes between plant lifetime (40 years) and investment cycle (20 years). Choose plant lifetime.
+electric arc furnace,electricity-input,2020,0.6395,MWh_el/t_steel,2020,"Mission Possible Partnership (2022): Steel Model (https://github.com/missionpossiblepartnership/mpp-steel-model/blob/9eca52db92bd2d9715f30e98ccaaf36677fdb516/mppsteel/data/import_data/Technology%20Business%20Cases.csv, accessed: 2022-12-03).",Based on process ‘EAF’. 
+electric arc furnace,hbi-input,2020,1,t_hbi/t_steel,2020,-,Assume HBI instead of scrap as input.Scrap would require higher input (in tonnes) as steel content is lower.
+shipping fuel methanol,fuel,2020,72,EUR/MWh_th,2020,"Based on (source 1) Hampp et al (2022), https://arxiv.org/abs/2107.01092, and (source 2): https://www.methanol.org/methanol-price-supply-demand/; both accessed: 2022-12-03.",400 EUR/t assuming range roughly in the long-term range for green methanol (source 1) and late 2020+beyond values for grey methanol (source 2).
+shipping fuel methanol,CO2 intensity,2020,0.2482,t_CO2/MWh_th,2020,-,Based on stochiometric composition.
```

Comment:
```suggestion
methanol,fuel,2020,72,EUR/MWh_th,2020,"Based on (source 1) Hampp et al (2022), https://arxiv.org/abs/2107.01092, and (source 2): https://www.methanol.org/methanol-price-supply-demand/; both accessed: 2022-12-03.",400 EUR/t assuming range roughly in the long-term range for green methanol (source 1) and late 2020+beyond values for grey methanol (source 2).
methanol,CO2 intensity,2020,0.2482,t_CO2/MWh_th,2020,-,Based on stochiometric composition.
```

Reply:
I don't know. It would be a bit biased, but should be fine I guess?

Reply:
Would also need to rename them in release notes.